diff -ru cobbler-3.3.3.orig/setup.py cobbler-3.3.3/setup.py
--- cobbler-3.3.3.orig/setup.py	2023-12-21 08:52:21.806069132 +0300
+++ cobbler-3.3.3/setup.py	2023-12-21 16:56:19.127024288 +0300
@@ -14,7 +14,6 @@
 from distutils.command.build import build as _build
 from configparser import ConfigParser
 from setuptools import find_packages
-from sphinx.setup_command import BuildDoc
 
 import codecs
 from coverage import Coverage
