--- a/gschem/src/gschem_compselect_dockable.c	2019-12-02 22:29:26.381624392 -0500
+++ b/gschem/src/gschem_compselect_dockable.c	2019-12-02 22:30:59.052424183 -0500
@@ -1699,10 +1699,10 @@
   if (compselect_is_tiled (compselect))
     return;
 
-  gboolean *size_allocated;
+  gboolean *size_allocated = NULL;
   const gchar *key;
   gint default_height;
-  GtkPaned *vpaned;
+  GtkPaned *vpaned = NULL;
 
   if (widget == compselect->preview_content) {
     size_allocated = &compselect->preview_size_allocated;
--- a/gschem/src/gschem_dockable.c	2019-12-02 22:31:39.420335533 -0500
+++ b/gschem/src/gschem_dockable.c	2019-12-02 22:34:28.739952678 -0500
@@ -1550,7 +1550,7 @@
                       GTK_CHECK_MENU_ITEM (check_menu_item));
 
   GschemDockableState current_state = gschem_dockable_get_state (dockable);
-  GschemDockableState target_state;
+  GschemDockableState target_state = current_state;
 
   if (check_menu_item == dockable->dock_left_item)
     target_state = GSCHEM_DOCKABLE_STATE_DOCKED_LEFT;
--- a/gschem/src/o_net.c	2019-12-02 22:36:46.971642601 -0500
+++ b/gschem/src/o_net.c	2019-12-02 22:37:10.675590708 -0500
@@ -82,7 +82,7 @@
   GList *object_list, *iter1, *iter2;
   OBJECT *o_current;
 
-  int *current_rules;
+  int *current_rules = NULL;
   /* badness values       {OVERWRITE, ORTHO, CONTINUE} */
   const int pin_rules[] = {100, 50, 0};
   const int bus_rules[] = {90, 0, 40};

